sdk/tables/azure-data-tables/tests/test_table_batch.py

```diff
@@ -163,7 +163,7 @@ def test_inferred_types(self):
         entity.test5 = EntityProperty(u"stringystring")
         entity.test6 = EntityProperty(3.14159)
         entity.test7 = EntityProperty(100)
-        entity.test8 = EntityProperty(10, EdmType.INT32)
+        entity.test8 = EntityProperty(2 ** 33)
```

Comment:
I would not have removed the `EntityProperty(10, EdmType.INT32)`, but instead add a test9 (more tests is better :)).

You still want to test that an explicit EdmType.INT32 is kept correctly